gslib/commands/perfdiag.py

```diff
@@ -250,18 +248,19 @@
 
 
 <B>NOTE</B>
-  The ``perfdiag`` command runs a series of tests and collects system information,
+  The ``perfdiag`` command runs a series of tests that collects system information,
   such as the following: 
   
   + Records requester's IP address.
   + Executes DNS queries to Google servers and collects the results.
   + Collects network statistics information from the output of ``netstat -s`` and
     evaluates the BIOS product name string.
-  + If configured, attempts to connect to the requester's proxy server
+  + If one is configured, attempts to connect to the requester's proxy server
```

Comment:
Maybe: If a proxy server is configured, attempts to connect to it to look up .....